src/assets/custom/js/websiteNavigation.test.js

```diff
@@ -235,14 +265,19 @@ function createMockSubMenu(subMenuID, subMenuToggleID) {
   );
   subMenuToggleProxy.classList.add(SUB_MENU_TOGGLE_PROXY_CLASS);
 
-  window.document.body.appendChild(subMenu);
+  // window.document.body.appendChild(subMenu);
```

Comment:
and this?

Reply:
Good spot! https://github.com/codurance/site/pull/1933/commits/c96f603e23db76226c321fa87032f41538b0218f